crates/store/re_types/definitions/rerun/archetypes/asset3d.fbs

```diff
@@ -33,4 +33,12 @@ table Asset3D (
   /// If omitted, the viewer will try to guess from the data blob.
   /// If it cannot guess, it won't be able to render the asset.
   media_type: rerun.components.MediaType  ("attr.rerun.component_recommended", nullable, order: 2000);
+
+  // --- Optional ---
+
+  /// An optional color for each vertex.
+  vertex_colors: [rerun.components.Color] ("attr.rerun.component_optional", nullable, order: 3100);
```

Comment:
Albedo factor makes sense to me because it can be easily applied to all materials inside a mesh format.
However, vertex color manipulation is pretty unclear to me: A model file has many submeshes and it's really hard to specify how the vertex indices from the vertex colors map to any of the vertices inside the models.
To make matters worse this depends a lot on the concrete model loader: What order are the submeshes in, are all submeshes left as is or can it happen that they flatten out, etc.

Due to those difficulties I'd rather not expose this at all, since it sounds like this might be a constant source of pain.
That is unless we find a way to clearly specify what this field does under all circumstances and make it a mesh/model file independent postprocessing step.